conf.yaml

```diff
@@ -4,6 +4,12 @@ paths:
     branch_tracker: html/branches.yaml
 
 repos:
+    stack:
+        url:        https://github.com/elastic/stack-docs.git
+        current:    master
```

Comment:
This upgrade guide is going to apply to a specific version for every release. Do we maybe want to build `current` from a 5.0 release for now?